lib/anagrams.rb

```diff
@@ -0,0 +1,28 @@
+class Anagrams
+  RESULT = {}
+
+  def initialize(filename)
+    @filename = filename
+  end
+
+  def read
```

Comment:
@StefanGajic not the best method name. First of there are many Ruby core methods named `read`. Second, `Anagrams#read` implies that it will read some anagrams - which it won't.

Better names would be: `read_dictionary` (or wordlist), `load_wordlist`, `parse_dictionary`, or similar. (you don't have to choose one of these, they were only first ones on my mind).